HomeWorks/src/main/java/com/epam/spring/homework2/beans/BeanB.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.spring.homework2.beans;
+
+import com.epam.spring.homework2.beans.validator.FieldsValidator;
+import com.epam.spring.homework2.beans.validator.MyValidator;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BeanB implements MyValidator {
+
+    @Value("${beanB.name}")
```

Comment:
This is not a good approach. Please rework:
- make class immutable
- pass values from properties files in `@Bean `method parameters in configuration class

Apply these changes to all relevant classes